### PR TITLE
Updated version numbers

### PR DESCRIPTION
**Pull request by TimothyLopas**

### CODE REVIEW

File: packages/windows/src/RPA/Windows/__init__.py [hunk 1]
```diff
@@ -41,10 +41,10 @@ class Windows(WindowsElementsMixin, DynamicCore):
         channels:
```
Comment (cmin764) [inline packages/windows/src/RPA/Windows/__init__.py:41]: I would refactor from above this as well:

From
```
Ultimately this library will replace current library ``RPA.Desktop.Windows`` if seen as approriate.
```

To (or similar)
```
This library replaces the old deprecated ``RPA.Desktop.Windows``, so please use ``RPA.Windows`` instead.
```

File: packages/windows/src/RPA/Windows/__init__.py [hunk 2]
```diff
@@ -41,10 +41,10 @@ class Windows(WindowsElementsMixin, DynamicCore):
...
           - conda-forge
         dependencies:
-          - python=3.7.5
-          - pip=20.1
+          - python=3.9.13
+          - pip=22.1.2
           - pip:
-            - rpaframework-windows==2.0.0
+            - rpaframework-windows==6.2.0
```
Comment (cmin764) [inline packages/windows/src/RPA/Windows/__init__.py:47]:
```suggestion
            - rpaframework-windows==6.2.0  # just the Windows library (`rpaframework` alone includes it)
```

**Review by cmin764 — APPROVED** [review]
LGTM!

Left a few related improvements, feel free to merge afterwards without a release note as this is just documentation. (that automatically builds on _master_)

File: packages/windows/src/RPA/Windows/__init__.py (hunk 1 above)
Comment (TimothyLopas) [reply packages/windows/src/RPA/Windows/__init__.py:41]: Change made

File: packages/windows/src/RPA/Windows/__init__.py (hunk 2 above)
Comment (TimothyLopas) [reply packages/windows/src/RPA/Windows/__init__.py:47]: change made

**Pull request merged**